gulpfile.js

```diff
@@ -527,7 +527,18 @@ const hbsHelpers = {
         return arr[i][resultField];
       }
     }
-  }
+  },
+  anyStartsWith: function (string, object) {
```

Comment:
This is an ok solution for the particular case, however it's not generic. This one implies that the "object" is really an object and not an Array, primitive, etc. The caller might (intentionally or by mistake) provide something else than the Object.

Since it's an overhead to handle every possible type, it's ok to only currently handle the "Object" (and add a TODO for the other types), but at least there should be some type checking then, if the provided parameter is really an Object.

Reply:
Good point. Check is the fix matches what you had in mind.